modules/appeals_api/app/models/appeals_api/notice_of_disagreement.rb

```diff
@@ -177,6 +193,7 @@ def mpi_veteran
 
     def email_identifier
       return { id_type: 'email', id_value: email } if email.present?
+      return { id_type: 'email', id_value: email_v2 } if email_v2.present?
```

Comment:
Updating `email` as suggested above would get rid of the need for this check, too.

Reply:
V1 form data will expect `email` and V2 will expect `email_v2`. The attribute name has changed from `emailAddressText` to `email`. This is how it is handled throughout. It will change once we upgrade our email handling.

Reply:
oh. derp. just saw your message above.